cem/test/fixtures/cc-test-component.js

```diff
@@ -0,0 +1,48 @@
+import { LitElement } from 'lit-element';
+
+/**
+ * @typedef {import('./cc-test-component.types.js').Foo} Foo
+ * @typedef {import('./cc-test-component.types.js').Bar} Bar
+ * @typedef {import('./cc-test-component.types.js').TheInterface} TheInterface
+ * @typedef {import('./cc-test-component.types.js').TheType} TheType
+ * @typedef {import('./cc-test-component.types.js').TupleFoo} TupleFoo
+ * @typedef {import('./cc-test-component.types.js').TupleBar} TupleBar
+ * @tyedef {import('./cc-test-component.types.js').TupleBar} TupleBar
```

Comment:
If it's supposed to be ignored, I would use it with something else than `TupleBar` so we can be sure it's ignored.